app/javascript/onboarding/Onboarding.jsx

```diff
@@ -69,16 +70,18 @@ export default class Onboarding extends Component {
     const { currentSlide } = this.state;
     const { communityConfig } = this.props;
     return (
-      <main
-        className="onboarding-body"
-        style={
-          communityConfig.communityBackground && {
-            backgroundImage: `url(${communityConfig.communityBackground})`,
+      <Focustrap>
+        <main
+          className="onboarding-body"
+          style={
+            communityConfig.communityBackground && {
+              backgroundImage: `url(${communityConfig.communityBackground})`,
+            }
           }
-        }
-      >
-        {this.slides[currentSlide]}
-      </main>
+        >
+          {this.slides[currentSlide]}
+        </main>
+      </Focustrap>
```

Comment:
```suggestion
      </FocusTrap>
```
(as per above comment)